src/commands/deployment/dashboard.ts

```diff
@@ -5,7 +5,7 @@ import { Config } from "../../config";
 import { getRepositoryName } from "../../lib/gitutils";
 import { exec } from "../../lib/shell";
 import { logger } from "../../logger";
-import { validatePrereqs } from "../infra/validate";
+import { validatePrereqs } from "./../init";
```

Comment:
maybe these should be `../../init`?

Reply:
I tested that the path `./../init` works, but also have condensed it `../init` now.